src/platform/web.rs

```diff
@@ -109,8 +109,8 @@ pub trait EventLoopExtWebSys {
 
     /// Initializes the winit event loop.
     ///
-    /// Unlike `run`, this returns immediately, and doesn't throw an exception in order to
-    /// satisfy its `!` return type.
+    /// Unlike `run()`, this returns immediately, and doesn't throw an exception in order to
```

Comment:
Can we link here to run as well?

Reply:
I might have not done that because of the above, but sure. Will need to add more rigorous checks for `exception-handling` anyway.

Reply:
Done. And in light of #3076 I have changed the footnotes to _always_ be emitted, so that readers on WASM _also_ get the same information (right in-place) that the reference is only available on their platform (with a specific feature).